### PR TITLE
Release 3.0.0-rc.3

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.0.0-rc.3 (2018-11-20)
+
+- Fixed colorpicker component firing the `valueChanges` event too many times when a form group is updated. [blackbaud/skyux-colorpicker#5](https://github.com/blackbaud/skyux-colorpicker/pull/5)
```
Comment (Blackbaud-AlexKingman) [inline CHANGELOG.md]: should that say "from firing" ?

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Uh. That probably make more sense.  One second

Comment (Blackbaud-SteveBrush) [reply CHANGELOG.md]: "Fixed (thing) to..."

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: @Blackbaud-SteveBrush check the updated string. (The above still shows the old one)

`Fixed colorpicker component to stop firing the `valueChanges` event multiples times when a form group is updated. `

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 3.0.0-rc.3 (2018-11-20)
+
+- Fixed colorpicker component to stop firing the `valueChanges` event multiples times when a form group is updated. [blackbaud/skyux-colorpicker#5](https://github.com/blackbaud/skyux-colorpicker/pull/5)
```
Comment (Blackbaud-SteveBrush) [inline CHANGELOG.md]: "multiples"

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Sigh. Fixed

**Pull request merged**